### DISCUSSION TITLE
Similar packages(Uno.Xamarin.Forms.Platform) available for MAUI

### DISCUSSION THREAD

**Discussion by Natpraveen** [unoplatform/uno#9881] [Q&A] [answered]
Will "Uno.Xamarin.Forms.Platform" plugin extend its support  MAUI. We are planning to migrate our existing product to MAUI and are also looking to support web.

Is there any plans to support  MAUI?

**Comment by jeromelaban** [comment, marked as answer]
Please see [this discussions](https://github.com/unoplatform/uno/discussions/8163#discussioncomment-2230829). If you have additional questions, you can contact us through info@platform.uno.
